server/settings/loaders.py

```diff
@@ -229,7 +252,9 @@ class LoadersModel(BaseSettingsModel):
         "vrayproxy": [255, 150, 12, 1.0],
         "vrayscene_layer": [255, 150, 12, 1.0],
         "yeticache": [99, 206, 220, 1.0],
-        "yetiRig": [0, 205, 125, 1.0]
+        "yetiRig": [0, 205, 125, 1.0],
+        "oxcache": [156, 234, 195, 1.0],
+        "oxrig": [206, 234, 195, 1.0]
```

Comment:
Technically your code isn't wrong and this is cosmetics only. But it just looks 'dangerous' with the below "commented" out still.
```suggestion
        "oxrig": [206, 234, 195, 1.0],
```

Also. We should really start removing those comments in another PR.